Source/Mosa.DeviceDriver/Mosa.DeviceDriver.csproj

```diff
@@ -2,12 +2,13 @@
   <PropertyGroup>
     <TargetFramework>net6.0</TargetFramework>
     <DebugType>full</DebugType>
-    <LangVersion>8.0</LangVersion>
+    <LangVersion>9.0</LangVersion>
     <OutputPath>..\..\bin\</OutputPath>
     <AppendTargetFrameworkToOutputPath>false</AppendTargetFrameworkToOutputPath>
     <DisableStandardFrameworkResolution>true</DisableStandardFrameworkResolution>
     <DisableImplicitFrameworkReferences>true</DisableImplicitFrameworkReferences>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+    <NuspecFile>Mosa.DeviceSystem.nuspec</NuspecFile>
```

Comment:
May not work within the project due to version number issue. 

This needs to be double checked first.

Reply:
What do you mean by version number?